src/NRules/NRules.Fluent/Dsl/QueryExtensions.cs

```diff
@@ -28,7 +41,8 @@ public static IQuery<TFact> Match<TFact>(this IQuery query, params Expression<Fu
         /// <param name="source">Query expression builder.</param>
         /// <param name="predicates">Filter expressions.</param>
         /// <returns>Query expression builder.</returns>
-        public static IQuery<TSource> Where<TSource>(this IQuery<TSource> source, params Expression<Func<TSource, bool>>[] predicates)
+        public static IQuery<TSource> Where<TSource>(this IQuery<TSource> source,
+            params Expression<Func<TSource, bool>>[] predicates)
```

Comment:
Can you please undo this non-functional reformatting. I have ReSharper line length configured at 180, I'll put that into the contrib guide

Reply:
Done and made the change to config. Oops